src/Microsoft.ML.StandardTrainers/Standard/LogisticRegression/MulticlassLogisticRegression.cs

```diff
@@ -31,6 +31,10 @@
     "Multiclass LR Executor",
     MaximumEntropyModelParameters.LoaderSignature)]
 
+[assembly: LoadableClass(typeof(LinearMulticlassModelParameters), null, typeof(SignatureLoadModel),
+    "Multiclass LR No Prob",
```

Comment:
>No Prob [](start = 19, length = 7)

should we say  "no calibration" or "raw scores" or "logits" ?

Reply:
I'm afraid we have limit in 26 chars for this string, and no calibration doesn't fit.
I'm fine with `Raw score`, but I would prefer to @wschin  comment on this one, since he was one who split them apart

Reply:
Multiclass LR No Calib",